### ISSUE TITLE
Program failure: fatal: token not implemented: kw_var

### ISSUE THREAD

**Issue by cahian** [quick-lint/quick-lint-js#352]
In the following code:
```javascript
var a =
var b = ""
```
Is received:
```
../src/parse.cpp:491: fatal: token not implemented in parse_expression: kw_var on line 2 column 1
[1]    436888 illegal hardware instruction  quick-lint-js file.js
```

**Comment by cahian** [comment]
```sh
$ sudo strace -p152610 -s9999 -e write                                   
[sudo] password for cahian: 
strace: Process 152610 attached
write(2, "/home/cahian/Repositories/quick-lint-js/src/parse.cpp:502: fatal: token not implemented in parse_primary_expression: kw_var", 123) = 123
write(2, " on line 16 column 1", 20)    = 20
write(2, "\n", 1)                       = 1
--- SIGABRT {si_signo=SIGABRT, si_code=SI_TKILL, si_pid=152610, si_uid=1000} ---
write(2, "/home/cahian/Repositories/quick-lint-js/src/parse.cpp:502: fatal: token not implemented in parse_primary_expression: kw_var", 123) = 123
write(2, " on line 16 column 1", 20)    = 20
write(2, "\n", 1)                       = 1
--- SIGSEGV {si_signo=SIGSEGV, si_code=SI_KERNEL, si_addr=NULL} ---
write(2, "/home/cahian/Repositories/quick-lint-js/src/parse.cpp:502: fatal: token not implemented in parse_primary_expression: kw_var", 123) = 123
write(2, " on line 16 column 1", 20)    = 20
write(2, "\n", 1)                       = 1
+++ exited with 127 +++
```

**Comment by dav000** [comment]
#412 resolves this issue

**Comment by cahian** [comment]
You're right. I tested. If you want, you can [link the pull request to this issue].

[link the pull request to this issue]: https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

**Comment by cahian** [comment]
Pull request #412 has been merged, so this issue is resolved.
